Fix some translations

Scirese opened the pull request:
1 ：I think "GrassClipper" is a proper noun，its inappropriate to translate it literally.
2："Play(游玩)" is not "launch(启动)".
3：“Genshin Impact Game（原神）” is not translated.
4："killswitchSubtitle" is completely wrong and seems like machine translate.
5：“set” here is preferably translated as "选择(select)"
6: Change some expressions of "folderNotSet" to let it looks clearer

Scirese commented:
These errors make me to think, if is this file translated by machine?

nuoxianCN commented:
I think this translation document is reasonable. The only drawback is that I can't understand the meaning of the paragraph of kill switch subtitle. 不要吹毛求疵

Scirese commented:
> I think this translation document is reasonable. The only drawback is that I can't understand the meaning of the paragraph of kill switch subtitle. 不要吹毛求疵

I fixed it and added some my own translation that I think is reasonable lol.
This comment has no offensive at all bro

nuoxianCN commented:
okey
